nuxeo-wopi/src/test/java/org/nuxeo/wopi/TestWOPIService.java

```diff
@@ -85,6 +86,9 @@ public void testGetActionURL() {
                 wopiService.getActionURL(blob, "view"));
         assertEquals("https://excel.officeapps-df.live.com/x/_layouts/xlviewerinternal.aspx?edit=1&IsLicensedUser=1&",
                 wopiService.getActionURL(blob, "edit"));
+        blob.setFilename("file.xls");
+        assertEquals("https://excel.officeapps-df.live.com/x/_layouts/ExcelConvertAndEdit.aspx?IsLicensedUser=1&",
+                wopiService.getActionURL(blob, "convert"));
```

Comment:
We could use `Constants#ACTION_CONVERT` here (well, same for the above actions, my bad)